stats/models/match.py

```diff
@@ -1,4 +1,6 @@
-class Match():
+from django.db import models
+
+class Match(models.Model):
     def __init__(self, username, match_data):
         self.id = match_data["metadata"]["matchId"]
```

Comment:
possible key not found exception.  You should try/catch and return in the case the id doesn't exist